### PR TITLE
added deadlettersinkuri set rules as must to brokers and channels spec

### LINKED ISSUE #83: Add Broker's status.deadLetterSinkURI behavior to spec
The [Broker lifecycle specification](https://github.com/knative/specs/blob/main/specs/eventing/control-plane.md#broker-lifecycle) doesn't include the desired behavior of the status.deadLetterSinkURI (as the Trigger does).
It should be added to it's spec (as optional at least)

### PR DESCRIPTION
**Pull request by gabo1208**
Fixes #83  and #84

### CODE REVIEW

**Review by evankanderson — COMMENTED** [review]
This looks like it was an oversight based on the 406-419 text in "Event Delivery" for `deadLetterSink`.

I'm +1 on this.

File: specs/eventing/control-plane.md [hunk 1]
```diff
@@ -134,6 +134,10 @@ an associated Trigger destination which does not currently have a `true`
 `Ready` condition, including events received by the Broker before the Trigger
 was created.
 
+If the Broker's `spec.delivery.deadLetterSink` field it set, it MUST be
+resolved to a URI and reported in `status.deadLetterSinkUri` in the same manner
+as the `status.address.url` field before setting the `Ready` condition to `true`.
```
Comment (evankanderson) [inline specs/eventing/control-plane.md:139]:
```suggestion
resolved to a URI and reported in `status.deadLetterSinkUri` before setting the `Ready` condition to `true`.
```

Comment (evankanderson) [reply specs/eventing/control-plane.md:139]: The reference to `status.address.url` didn't seem to help anyone understand the pattern.

Comment (duglin) [inline specs/eventing/control-plane.md:139]: Can we tweak the wording slightly:
```
If the Broker's `spec.delivery.deadLetterSink` field is set, it MUST be
resolved to a URI and reported in `status.deadLetterSinkUri` before
setting the `Ready` condition to `true`.
```
Just easier for me to read ;-)

File: specs/eventing/control-plane.md [hunk 2]
```diff
@@ -198,6 +202,10 @@ events to an associated Subscription which does not currently have a `true`
 `Ready` condition, including events received by the Channel before the
 `Subscription` was created.
 
+If the Channel's `spec.delivery.deadLetterSink` field it set, it MUST be
```
Comment (evankanderson) [inline specs/eventing/control-plane.md:205]: s/it/is/

File: specs/eventing/control-plane.md [hunk 3]
```diff
@@ -198,6 +202,10 @@ events to an associated Subscription which does not currently have a `true`
...
+resolved to a URI and reported in `status.deadLetterSinkUri` in the same manner
+as the `status.address.url` field before setting the `Ready` condition to `true`.
```
Comment (duglin) [inline specs/eventing/control-plane.md:207]: And:
```
If the Channel's `spec.delivery.deadLetterSink` field is set, it MUST be
resolved to a URI and reported in `status.deadLetterSinkUri` before
setting the `Ready` condition to `true`.
```

Note:  s/it/is/  on both

**Comment by gabo1208** [comment]
Thanks <3 @evankanderson and @duglin fixed the typos and grammar mistakes :)!

**Pull request merged**